packages/mantine/src/components/modal-wizard/ModalWizard.tsx

```diff
@@ -143,7 +143,7 @@ export const ModalWizard: ModalWizardType = ({
         >
             {currentStep.props.showProgressBar && <Progress color="teal" size="lg" value={getProgressMemo} />}
             {currentStep}
-            <StickyFooter borderTop py={null} px={null} pt="md">
+            <StickyFooter py={0} px={0} pt="sm" borderTop>
```

Comment:
There are others to change on line 133 and 134.

You could bump Mantine to make sure we catch them all

Reply:
I changed line 133 & 134. And I also bumped mantine to 5.8.3!